airflow/www/forms.py

```diff
@@ -33,7 +33,7 @@ class DateTimeWithNumRunsForm(Form):
     # Date time and number of runs form for tree view, task duration
     # and landing times
     base_date = DateTimeField(
-        "Anchor date", widget=DateTimePickerWidget(), default=datetime.utcnow())
+        "Anchor date", widget=DateTimePickerWidget(), default=timezone.utcnow())
```

Comment:
I realise we're probably at the "making it work" stage, but the forms might want to use DatePickers etc in local time.

Reply:
I consider that out of scope for the change I am doing.